x/multisig/codec.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package multisig;
+
+message Contract {
+    // contract's address
+    bytes address = 1;
+    // addresses to control it
+    repeated bytes sigs = 2;
+    // threshold needed to sign to activate it
+    int64 activationThreshold = 3;
+    // threshold needed to sign to change it
+    int64 changeThreshold = 4;
+}
+
+message CreateContractMsg {
+    // contract's address
+    bytes address = 1;
```

Comment:
Let the system decide... No need to let the user select one

Reply:
I would add a ModifyContractMsg that looks like the current CreateContractMsg definition (with bytes address)